apps/web-giddh/src/app/contact/contact.component.html

```diff
@@ -3,7 +3,7 @@
 <hamburger-menu [pageHeading]="'Vendor'" *ngIf="activeTab === 'vendor'"></hamburger-menu>
 <tabset #staticTabs class="custom-tabs" [ngClass]="{'hide-header': activeTab === 'vendor'}">
     <!-- region Customer/Vendor -->
-    <tab heading="Customer" (selectTab)="tabSelected(activeTab === 'vendor' ? 'vendor' : 'customer')" [active]="activeTab !== 'aging-report'">
+    <tab heading="Customer" [id]="(activeTab !== 'aging-report')?'giddh-page-heading': ''" (selectTab)="tabSelected(activeTab === 'vendor' ? 'vendor' : 'customer')" [active]="activeTab !== 'aging-report'">
```

Comment:
Use spacing in condition for better readability. Please do this in all the files where the conditions are added
```suggestion
    <tab heading="Customer" [id]="(activeTab !== 'aging-report') ? 'giddh-page-heading' : ''" (selectTab)="tabSelected(activeTab === 'vendor' ? 'vendor' : 'customer')" [active]="activeTab !== 'aging-report'">
```